src/main/java/frc/robot/RobotContainer.java

```diff
@@ -58,30 +64,101 @@
-    private final Vision hubCam = new Vision(
-            new VisionIOPhotonVision(
-                    VisionIOPhotonVision.HUB_CAMERA_NAME,
-                    VisionIOPhotonVision.HUB_CAMERA_HEIGHT_METERS,
-                    VisionIOPhotonVision.HUB_CAMERA_DEGREES_HORIZ));
+    private DriveBase drive;
+    private Climber climber;
+    private Intake intake;
+    private VarFlyWheel varFlyWheel;
+    private Transversal transversal;
+    private Uptake uptake;
+    private ColorSensor colorSensor;
+    private Vision hubCam;
     private UsbCamera camera;
 
     /**
      * The container for the robot. Contains subsystems, OI devices, and commands.
      */
     public RobotContainer() {
+
+        // Initialize Subsystems
+        configureSubsystems();
+
         // Configure the button bindings
         configureButtonBindings();
-        camera = CameraServer.startAutomaticCapture();
-        camera.setConnectionStrategy(VideoSource.ConnectionStrategy.kKeepOpen);
+    }
+
+    private void configureSubsystems() {
+        switch (Constants.bot) {
+            case ROBOT_2022_COMP:
+                drive = new DriveBase(new DriveIOFalcon());
+                climber = new Climber(
+                        new ClimberIOFalcon(),
+                        new ClimberPistonIO());
+                intake = new Intake(new IntakeIOTalonSRX());
+                varFlyWheel = new VarFlyWheel(new VarFlyWheelIOFalcon());
+                transversal = new Transversal(new TransversalIOSparkMax());
+                uptake = new Uptake(new UptakeIOSparkMax());
+                colorSensor = new ColorSensor(new ColorSensorIORevV3());
+                hubCam = new Vision(
+                        new VisionIOPhotonVision(
+                                VisionIOPhotonVision.HUB_CAMERA_NAME,
+                                VisionIOPhotonVision.HUB_CAMERA_HEIGHT_METERS,
+                                VisionIOPhotonVision.HUB_CAMERA_DEGREES_HORIZ));
+                camera = CameraServer.startAutomaticCapture();
+                camera.setConnectionStrategy(VideoSource.ConnectionStrategy.kKeepOpen);
+                break;
+            case ROBOT_2022_PRACTICE:
+                drive = new DriveBase(new DriveIOFalcon());
+                hubCam = new Vision(
+                        new VisionIOPhotonVision(
+                                VisionIOPhotonVision.HUB_CAMERA_NAME,
+                                VisionIOPhotonVision.HUB_CAMERA_HEIGHT_METERS,
+                                VisionIOPhotonVision.HUB_CAMERA_DEGREES_HORIZ));
+                break;
+            default:
+                System.out.println("No robot selected.");
+                break;
+        }
+
+        if (drive == null) {
+            drive = new DriveBase(new DriveIO() {
+            });
+        }
+        if (climber == null) {
+            climber = new Climber(new ClimberIO() {
+            }, null);
```

Comment:
This line needs to be changed to remove the null. If we call any climber tilt commands in TeleOp while using practice bot, we will get a NullReferenceException